frontend/src/hooks/helpers.ts

```diff
@@ -51,188 +48,208 @@ export const getAvailabilityValue = (
   };
 };
 
+const renderAvailability = (
+  personnel: Personnel,
+  filterValues: DashboardFilters,
+) => {
+  if (
+    filterValues.availabilityType &&
+    personnel?.availability &&
+    personnel?.availability?.length > 0
+  ) {
+    return getAvailabilityValue(
+      filterValues.availabilityType,
+      filterValues.availabilityDates,
+      personnel.availability,
+    );
+  } else if (!personnel.availability) {
+    return { availability: '' };
+  } else if (personnel.availability.length > 0) {
+    return {
+      availability:
+        AvailabilityTypeName[
+          personnel.availability[0].availabilityType as keyof typeof AvailabilityType
+        ],
+    };
+  }
+};
+
 export const renderCells = (
   personnel: Personnel,
   filterValues: DashboardFilters,
+  route: Route,
 ) => {
-  const columns = {
+  const cells = {
     name: {
       key: DashboardColumns.NAME,
       columnName: DashboardColumns.NAME,
       value: `${personnel.lastName?.toUpperCase()},  ${personnel.firstName}`,
-      className: tableClass(DashboardColumns.NAME, ''),
     },
 
     region: {
       key: DashboardColumns.REGION,
       columnName: DashboardColumns.REGION,
-      value: personnel.homeLocation.region,
-      className: tableClass(
-        DashboardColumns.REGION,
-        personnel.homeLocation?.region?.toLowerCase() ?? '',
-      ),
+      value: personnel.homeLocation?.region,
+    },
+
+    fireCentre: {
+      key: DashboardColumns.FIRE_CENTRE,
+      columnName: DashboardColumns.FIRE_CENTRE,
+      value: FireCentreName[personnel?.homeFireCentre?.fireCentre],
```

Comment:
Here, but also in general for this file, since personnel and homeFireCentre are nullished, do we need to define a value if anything is null?
i.e.
`FireCentreName[personnel?.homeFireCentre?.fireCentre] || 'Unknown'`